ecssd_agent.go

```diff
@@ -176,6 +176,43 @@ func getDNSHostedZoneId() (string, error) {
 	return "", err
 }
 
+func createARecord(hostName string, localIP string) error {
+	r53 := route53.New(session.New())
+	aRecordName := configuration.Hostname
+	// This API call creates a new DNS record for this service
+	params := &route53.ChangeResourceRecordSetsInput{
+		ChangeBatch: &route53.ChangeBatch{
+			Changes: []*route53.Change{
+				{
+					Action: aws.String(route53.ChangeActionCreate),
+					ResourceRecordSet: &route53.ResourceRecordSet{
+						Name: aws.String(aRecordName),
+						// It creates an A record with the IP of the host running the agent
+						Type: aws.String(route53.RRTypeA),
+						ResourceRecords: []*route53.ResourceRecord{
+							{
+								// priority: the priority of the target host, lower value means more preferred
+								// weight: A relative weight for records with the same priority, higher value means more preferred
+								// port: the TCP or UDP port on which the service is to be found
+								// target: the canonical hostname of the machine providing the service
```

Comment:
Nice Catch Ingmar!

I'll comb through the comments and make sure I didn't miss any other ones that need to be updated or deleted as well.